fabbwled-backend/src/main/resources/sections/109-144.yaml

```diff
@@ -0,0 +1,136 @@
+- number: 109
+  content:
+    - text: You are about to leave when you see a crowd gathered around a slim pasty-faced scholar at a card table. ‘By the Three Fortunes, but I’m hot tonight!’ he cries. It seems he is on a winning streak. You notice a couple of dodgy- looking ruffians watching the scholar carefully.
+    - choice:
+        - text: Wait and follow the scholar
+          then:
+            - turnTo: 540
+        - text: Return to the city centre
+          then:
+            - turnTo: 100
+
+- number: 111
+  content:
+    - text: Your knowledge of the arcane arts is too limited to help you here.
+    - choice:
+        - text: Fight your way to the golden net
+          then:
+            - turnTo: 121
+        - text: Swim back to Shadar Tor
+          then:
+            - turnTo: 35
+
+- number: 114
+  content:
+    - text: You are known as the priest who outwitted the storm demons to save Sul Veneris. You are shown in to see the chief administrator, who welcomes you freely. He can heal you of all lost Stamina points and cure you of a disease or poison free of charge. He cannot, however, lift a curse. ‘Whenever you need help, come to me!’ he says.
+    - text: When you are ready, turn to 100.
```

Comment:
Remove the turn to 100 in the text as it is already defined

Reply:
done